### PR TITLE
Combined navigation bar and list of bills

### LINKED ISSUE #47: Сombine navigation bar and list view.
Part of issue #3 

Depends on #42 #44 #43 #52.

Сombine navigation bar and list view.

### Mockup pictures:

When the list is not empty:
Screenshot 2023-05-23 at 02 05 10

### User perspective

- The screen should contain on the navigation bar and the list of bills.  
- The title "History of Operations" in the middle of the top;
- The "Add" button in the upper right corner.

- Under navigation bar should be scrollable list of bills described in the #43 . The button "Add" and a tap on the row with bill don't trigger any events. 

### Implementation details:

- take the list view that is the result of the #43. 
- take the adjusted navigation bar of the `HistoryOfOperationsView` that is the result of the #52. 
- as the result, the list view is called in the `NavigationStack` of the `HistoryOfOperationScreen`

```swift
struct HistoryOfOperationScreen: View {
   var body: some View {
        NavigationStack {
            BillsListView()

//...
}
```

### PR DESCRIPTION
**Pull request by levvaa**
### What?

Combined navigation bar with static list of bills. 

### Why?

Closes #47 

### How?

`BillListView` added into the `HistoryOfOperationsView`

### CODE REVIEW

**Review by dersim-davaod — CHANGES_REQUESTED** [review]
Please make the proper project structure.

As an example I'd suggest to put the `HistoryOfOperationView` into a separate group. Then please put view's components (`BillListView`, `BillRowView`, etc) into the `HistoryOfOperationView/Subviews` dir or something like that to show the hierarchy of views.

**Review by kirpichonok — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,3 +14,4 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Navigation bar for screen `History of Operations` [#52](https://github.com/ios-course/link-team-project/pull/52)
 - Bill list view for screen `History of Operations` [#43](https://github.com/ios-course/link-team-project/issues/43)
 - `BlackRoundedRectangleButton`, `CircleWithPlusButton` button styles [#60](https://github.com/ios-course/link-team-project/pull/60)
+- `HistoryOfOperationsView` contains a navigation bar and a list with static data [#47](https://github.com/ios-course/link-team-project/issues/47)
```
Comment (dersim-davaod) [inline CHANGELOG.md:17]: nit:

This is not an addition, but the update.

Comment (dersim-davaod) [inline CHANGELOG.md:17]:
```suggestion
- `HistoryOfOperationsView` containing a navigation bar and a list with static data [#47](https://github.com/ios-course/link-team-project/issues/47)
```

Comment (dersim-davaod) [inline CHANGELOG.md:17]: nit:

I'd suggest to make it more clear.

```suggestion
- `HistoryOfOperationsView` screen where user can see the static list of operations/bills and navigate to the `AddBill` screen [#47](https://github.com/ios-course/link-team-project/issues/47)
```

File: JourneysSquad/JourneysSquad.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -218,12 +215,31 @@
...
+				ABF7B5C32A20AB93006EB1EF /* DummyBillListViewModel.swift */,
 			);
 			path = ViewModel;
 			sourceTree = "<group>";
 		};
+		ABDE72162A279F53003810A8 /* HistoryOfOperationsView */ = {
+			isa = PBXGroup;
+			children = (
+				BA7BE3042A1DFEC8005D0395 /* HistoryOfOperationView.swift */,
+				ABDE72172A279FAA003810A8 /* HistoryOfOperationsSubviews */,
```
Comment (dersim-davaod) [inline JourneysSquad/JourneysSquad.xcodeproj/project.pbxproj:228]: nit:

`Subviews` would be enough, but that's fine.

```suggestion
				ABDE72172A279FAA003810A8 /* Subviews */,
```

**Review by dersim-davaod — APPROVED** [review]
LGTM, but consider minor improvements please:

File: JourneysSquad/JourneysSquad.xcodeproj/project.pbxproj (hunk 2 above)
Comment (levvaa) [reply JourneysSquad/JourneysSquad.xcodeproj/project.pbxproj:228]: Yeah! You right. Fixed.

File: CHANGELOG.md (hunk 1 above)
Comment (levvaa) [reply CHANGELOG.md:17]: Your version looks better. I used it!

Comment (levvaa) [reply CHANGELOG.md]: fixed. Used your suggestion below

**Pull request merged**